.changeset/heavy-elephants-nail.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/core-app-api': minor
+---
+
+# test
```

Comment:
🤔

Reply:
Oh, shoot, I must have fat fingered something, will get this fixed

Reply:
This has been cleaned up, sorry about that